docs/koji.md

```diff
@@ -0,0 +1,47 @@
+## Koji integration
+
+The `koji` pre-build plugin and the `koji_promote` exit plugin provide integration with [Koji](https://docs.pagure.org/koji/).
+
+## Pre-build plugin
+
+The `koji` pre-build plugin creates a yum repofile to allow instructions in the Dockerfile to have access to RPMs available from the appropriate Koji tag.
+
+## Exit plugin
```

Comment:
Do you think it's worth listing out a somewhat full json which includes all the attributes listed in this doc? Might be easier to visualize it all.